plasmapy/formulary/tests/test_plasma_frequency.py

```diff
@@ -0,0 +1,185 @@
+"""
+Module for testing functionality associated with calculating the
+plasma frequency.
+
+- `~plasmapy.formulary.parameters.plasma_frequency`
+- `~plasmapy.formulary.parameters.plasma_frequency_lite`
+- `~plasmapy.formulary.parameters.wp_`
+"""
+import astropy.units as u
+import numpy as np
+import pytest
+
+from astropy.constants.si import m_p
+from numba.extending import is_jitted
+
+from plasmapy.formulary.parameters import plasma_frequency, plasma_frequency_lite, wp_
+from plasmapy.particles import Particle
+from plasmapy.utils.pytest_helpers import assert_can_handle_nparray
+
+
+@pytest.mark.parametrize(
+    "alias, parent",
+    [(wp_, plasma_frequency)],
+)
+def test_aliases(alias, parent):
+    assert alias is parent
+
+
+class TestPlasmaFrequency:
+    """
+    Test class for `plasmapy.formulary.parameters.plasma_frequency`.
+
+    Note: Testing of `plasma_frequency_lite` is done in a separate test
+    class.
+    """
+
+    @pytest.mark.parametrize(
+        "bound_name, bound_attr",
+        [("lite", plasma_frequency_lite)],
+    )
+    def test_lite_function_binding(self, bound_name, bound_attr):
+        """Test expected attributes are bound correctly."""
+        assert hasattr(plasma_frequency, bound_name)
+        assert getattr(plasma_frequency, bound_name) is bound_attr
```

Comment:
Since this sort of binding test is going to be done for every lite function, a possibility for the future would be to create a base class for formulary tests that has this test (and other tests) built in.  That's out of scope for this PR, though.

Reply:
You read my mind!  I was thinking the same thing and have been mulling over a possible structure.  I might try to add it in the next lite PR.